3574/2/helpers/photo.rb

```diff
@@ -0,0 +1,19 @@
+require 'open-uri'
+require 'json'
+
+module Photo
```

Comment:
I'd name such modules with word Helpers
PhotoHelpers, LocationHelpers and so own